src/V3OrderParallel.cpp

```diff
@@ -1710,7 +1710,9 @@ class DpiImportCallVisitor final : public VNVisitor {
         if (nodep->dpiImportWrapper()) {
             if (nodep->dpiPure() ? !v3Global.opt.threadsDpiPure()
                                  : !v3Global.opt.threadsDpiUnpure()) {
-                m_hasDpiHazard = true;
+                m_hasDpiHazard = V3Config::getProfileData(nodep->cname()) == 0;
+                UINFO(9, "DPI wrapper '" << nodep->cname()
+                                         << "' has dpi hazard = " << m_hasDpiHazard << endl);
```

Comment:
Zero seems to now have special meaning, please document this in the cost member variable declaration.

Reply:
I'd added documentation above this `m_hasDpiHazard` assignment, as this special meaning is constrained only to DPI import wrappers, rather than all `AstCFunc` nodes.